test/test-MetadataStore.js

```diff
@@ -181,6 +186,42 @@ exports.test_async_insert_all = function*(assert) {
   assert.deepEqual(items[7], [3, 5]);
 };
 
+exports.test_get_oldest_insert = function*(assert) {
+  // insert metadata at 3 different times
+  for (let metadata of metadataFixture) {
+    yield sleep(3000);
```

Comment:
I'd say let it sleep for now. We can make this change if manual specifying `created_at` becomes ubiquitous. What do you think? @sarracini